connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/ExecutePlanHolder.scala

```diff
@@ -30,13 +31,20 @@ case class ExecutePlanHolder(
   val jobTag =
     "SparkConnect_" +
       s"User_${sessionHolder.userId}_Session_${sessionHolder.sessionId}_Request_${operationId}"
+  val events: RequestEvents = RequestEvents(this, new SystemClock())
 
   def interrupt(): Unit = {
     // TODO/WIP: This only interrupts active Spark jobs that are actively running.
     // This would then throw the error from ExecutePlan and terminate it.
     // But if the query is not running a Spark job, but executing code on Spark driver, this
     // would be a noop and the execution will keep running.
     sessionHolder.session.sparkContext.cancelJobsWithTag(jobTag)
+
+    // TODO: The above does not cancel the Connect query and it continues executing.
+    // events.postCanceled()
   }
 
+  def removeFromSessionHolder(): Unit = {
+    sessionHolder.removeExecutePlanHolder(operationId)
+  }
```

Comment:
I think you're not calling it anywhere? (unless sth didn't unfold for me)

Reply:
Agreed, removed